src/ResourceManager/AzureBatch/Commands.Batch/AzureRM.Batch.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '4.1.5'
+ModuleVersion = '5.0.0'
```

Comment:
@matthchr as mentioned offline, you can remove this file since we'll be using `Az.Batch.psd1`

Reply:
Deleted this file